app/views/notes/new.html.erb

```diff
@@ -8,6 +8,7 @@
       <%= form.govuk_error_summary %>
 
       <%= form.hidden_field :task_id %>
+      <%= form.hidden_field :task_identifier %>
```

Comment:
I do wonder if this is the cause of the empty string issue you have and we could avoid it.

When you are a new task note in the existing task list you have the `task_id` so I don't think we need line 10 at all.

In a simiar way, we can probably make the task identifier part of the path and get it from there. With a new note being added to a task at a path like:

`http://localhost:3000/projects/conversion/involuntary/63DE9634-EC09-455B-A118-3030EC391412/tasks/handover/notes/new`

Anyway, I think what is here is fine for now. I'd like to come back once we have the new task list in place and look at refactors like this one.